### ISSUE TITLE
Possibility of calculation profit

### ISSUE THREAD

**Issue by lotexteam** [InvoicePlane/InvoicePlane#732]
When we create product, we can set Purchase price, is it possible to automatically generate invoice for billing, where we can see profit?

**Comment by MediaElf** [comment]
Yes, to do that you need to infiltrate the database and then write the code in the corresponding php file

**Comment by spacecabbie** [comment]
Added as feature request issue closed since features are traced else where.
